articles/azure-monitor/snapshot-debugger/snapshot-debugger.md

```diff
@@ -48,11 +48,8 @@ The main process continues to run and serve traffic to users with little interru
 
 Snapshot collection is available for:
 
```

Comment:
this is now not focused on apps running those versions. by design?

Reply:
@AaronMaxwell